pkg/customizations/kickstart/kickstart.go

```diff
@@ -0,0 +1,46 @@
+package kickstart
+
+import "github.com/osbuild/images/pkg/customizations/users"
+
+type File struct {
+	Contents string
+}
+
+type OSTree struct {
+	OSName string
+	Remote string
+}
+
+type Options struct {
+	// Path where the kickstart file will be created
+	Path string
+
+	// Add kickstart options to make the installation fully unattended
+	Unattended bool
+
+	// Create a sudoers drop-in file for each user or group to enable the
+	// NOPASSWD option
+	SudoNopasswd []string
+
+	// Kernel options that will be apended to the installed system
+	// (not the iso)
+	KernelOptionsAppend []string
+
+	// Enable networking on on boot in the installed system
+	NetworkOnBoot bool
+
+	Language *string
+	Keyboard *string
+	Timezone *string
+
+	OSName string
```

Comment:
These are for OSTree, right? Why not use the OSTree struct defined above?

Reply:
I am looking at this and scratching my head because I'm sure that was the intention but I somehow went a different way halfway through.

Reply:
Alright, fixed.  Thanks for catching this!!